package.json

```diff
@@ -173,6 +174,9 @@
     "ts-node": "^9.1.1",
     "typescript": "^4.1.3"
   },
+  "optionalDependencies": {
```

Comment:
this can fail and it's fine, so if you have access to the repo you get the sauce, and if not, no biggie